docs/samples/Microsoft.ML.Samples/Dynamic/ConcatTransform.cs

```diff
@@ -1,8 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
thanks for taking care of the copyright too.

 Final nit, if you want to change the indent of using and just have them not indented at all.